modules/test/playwright/utils/util.ts

```diff
@@ -3,6 +3,36 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import * as fs from 'fs';
```

Comment:
Instead of having `util.ts` as a bucket of utils, could we have one util per file, each named like the exported function? Then we can add multiple tests per util, each in own file.

Reply:
I'm OK with separating things in smaller modules, but...

What about `createTempFile` and `readTempFile`? Those two functions need to share the `TMP_DIR` so they need to go in the same module... :grimacing: 

Can we use a domain based approach, for example? :thinking: 

Like `util/files.ts` and `util/random.ts`, or something like that...

Reply:
Since there are only 3 functions I'll pass the PR like this so that I don't have to wait for another day. We can refactor it in a future PR.

Reply:
I have finally refactored the utils as per the frontend guidelines. Will push the changes soon.